src/main/java/com/jwplayer/jwplatform/client/AdvertisingClient.java

```diff
@@ -0,0 +1,13 @@
+package com.jwplayer.jwplatform.client;
+
+public class AdvertisingClient{
+
+	private String path;
+	private String secret;
+	
+	public AdvertisingClient(String secret) {
```

Comment:
Please use static factory methods instead of public constructors. This is the pattern established in the current client.